x/ccv/consumer/keeper/relay.go

```diff
@@ -223,6 +236,40 @@ func (k Keeper) SendPackets(ctx sdk.Context) {
 // in conjunction with the ibc module's execution of "acknowledgePacket",
 // according to https://github.com/cosmos/ibc/tree/main/spec/core/ics-004-channel-and-packet-semantics#processing-acknowledgements
 func (k Keeper) OnAcknowledgementPacket(ctx sdk.Context, packet channeltypes.Packet, ack channeltypes.Acknowledgement) error {
+	if res := ack.GetResult(); res != nil {
+		if len(res) != 1 {
+			return fmt.Errorf("acknowledgement result length must be 1, got %d", len(res))
```

Comment:
Is this error message logged by IBC? If not, we should log it for better debugging.

Reply:
Yea it seems to be logged here https://github.com/cosmos/ibc-go/blob/000e9d20b15b3c6939cefc0e26daa4b64c1be8fb/modules/core/keeper/msg_server.go#L680